bundles/org.openhwgroup.corev.ide.toolchain.gnu.core/plugin.xml

```diff
@@ -90,7 +90,7 @@
                cleanCommand="${cross_rm} -rf"
                id="org.openhwgroup.corev.ide.toolchain.llvm.core.elf.release"
                name="%configuration.elf.release.name"
-               parent="cdt.managedbuild.config.gnu.base">
+               parent="cdt.managedbuild.config.llvm.base">

```

Comment:
This is very strange parent

Reply:
Seems I forgot that it was me who mixed all in one bundle. Please ignore my comment above.